src/store/emptyImageBuilderApi.ts

```diff
@@ -5,6 +5,24 @@ import { IMAGE_BUILDER_API } from '../constants';
 // initialize an empty api service that we'll inject endpoints into later as needed
 export const emptyImageBuilderApi = createApi({
   reducerPath: 'imageBuilderApi',
-  baseQuery: fetchBaseQuery({ baseUrl: IMAGE_BUILDER_API }),
+  baseQuery: fetchBaseQuery({
+    baseUrl: IMAGE_BUILDER_API,
+    paramsSerializer: (params) => {
```

Comment:
I'd like to have a note her about why there is a necessity to use the paramSerializer for the image builder API. Look at it as giving a hint for future us about why we're doing things. It reduces the risk of code becoming debt over time.

Reply:
ok, added your coments that you suggest me in the preview review